pages/homepage/homepage.html

```diff
@@ -0,0 +1,251 @@
+            <div class="burger nav-section-items">
+                <div class="line1"></div>
+                <div class="line2"></div>
+                <div class="line3"></div>
+            </div>
+
+            <div class="nav-logo nav-section-items logo">
+                <a href="#">
+                    <div class="navbar-logo flex flex-center">
+                        <h1>m<strong>U</strong>s<strong>I</strong>k</h1>
+                    </div>
+                </a>
+            </div>
+        </div>
+
+        <div class="nav-section">
+            <label class="search-bar nav-section-item-width50pc">
+                <span class="search-bar-btn" type="submit"><i class="fa fa-search"> </i></span>
+                <input class="search-bar-input" type="text" placeholder="Type to search" name="search">
+            </label>
+
+            <ul class="nav-icons list-style-none nav-section-item-width50pc">
+                <li class="list-inline-item">
+                    <a class="nav-icon-link link-no-style">
+                        <span class="btn-icon badge-container"><i class="fas fa-heart"></i>
+                            <span class="status-badge status-badge-number">4</span>
+                        </span>
+                        <span class="nav-icon-text">
+                            Wishlist
+                        </span>
+                    </a>
+                </li>
+                <li class="list-inline-item">
+                    <a class="nav-icon-link link-no-style">
+                        <span class="btn-icon badge-container"><i class="fas fa-shopping-cart"></i>
+                            <span class="status-badge status-badge-number">2</span>
+                        </span>
+                        <span class="nav-icon-text">
+                            Cart
+                        </span>
+                    </a>
+                </li>
+                <li class="list-inline-item hide-profile-mobile">
+                    <a class="nav-icon-link link-no-style">
+                        <span class="btn-icon badge-container"><i class="fas fa-user"></i>
+                        </span>
+                        <span class="nav-icon-text">
+                            Login
+                        </span>
+                    </a>
+                </li>
+            </ul>
+        </div>
+    </nav>
+</header>
+
+<main>
+        <div class='hero-image display-flex vertical-middle'>
+            <div class='text-center'>
+                <div>
+                    <h1 class='banner-title'>GRAB THE BEST DEALS</h1>
+                </div>
+                <div class="btn btn-solid-primary btn-lg-size"> Shop Now </div>
+            </div>
+        </div>
+
+        <div class="spacer-3rem"></div>
+        <div class="spacer-3rem"></div>
+
+        <div class='homepage-container'>
+
+            <h2 class='text-center feature-title'>Featured Products</h2>
+            <div class='title-underline'></div>
```

Comment:
<hr> tag can be used here just a suggestion for semantic.

Reply:
to wrap the featured product's title?